Add link to the troubleshoot section (#1503)
hyperledger/besu-docs: docs/public-networks/how-to/connect/manage-peers.md

The minimum number of peers is set by the `--Xp2p-peer-lower-bound` option, which also has a default of 25. If you reduce the `--max-peers` from the default, you must also set the `--Xp2p-peer-lower-bound` option to the same value or lower. For example, if you decrease `--max-peers` to 20, set `--Xp2p-peer-lower-bound` to 20 or lower.

> `--Xp2p-peer-lower-bound` is an early access option.

Author: Just making this consistent with the other EA options

Reviewer: this option has actually been deprecated and will be removed soon https://github.com/hyperledger/besu/pull/6501

Author: Thanks @macfarla! I think I will open a separate issue to correct

Author: Actually I will just add it to this PR

Author: So should I revert this change to just say the option is EA? I think the other section when `--Xp2p-peer-lower-bound` is mentioned is in the options page. Should I not add the deprecation note here and open an issue to fix both instances?

Reviewer 2: Yes, feel free to revert the change here and cover this in a separate PR.

:::

> 

Reviewer 2: ```suggestion
```

Please always check your "files changed" or run `git diff` before staging to make sure you're not committing accidental changes such as unnecessary line breaks.

Author: I edited the PR and thought I made this change. Just an error on my end

> The minimum number of peers is set by the `--Xp2p-peer-lower-bound` option, which also has a default of 25. If you reduce the `--max-peers` from the default, you must also set the `--Xp2p-peer-lower-bound` option to the same value or lower. For example, if you decrease `--max-peers` to 20, set `--Xp2p-peer-lower-bound` to 20 or lower.

Reviewer: can we remove this line since this option is deprecated

## Troubleshoot

> If you encounter issues with peering, refer to the [troubleshoot peering documentation](../../how-to/troubleshoot/peering.md) which provides guidance on identifying and resolving common problems that can occur during the peering process.

Reviewer 2: ```suggestion
If you encounter issues with peering, see the [troubleshoot peering documentation](../../how-to/troubleshoot/peering.md), which helps you identify and resolve common problems that can occur during the peering process.
```

If your nodes fail to connect, ensure the [required ports are open](configure-ports.md).

If your nodes are running in AWS, ensure you have the appropriate `SecurityGroups` configured to allow access to the required ports.

Check that the [enode URLs](../../concepts/node-keys.md#enode-url) specified for [bootnodes](../../../private-networks/how-to/configure/bootnodes.md) or [static nodes](static-nodes.md) match the enode URLs displayed when starting the remote nodes.

> If your node is having trouble peering, see the [troubleshoot peering documentation](../../how-to/troubleshoot/peering.md).

Reviewer 2: ```suggestion
If you still have peering issues, see the [troubleshoot peering documentation](../../how-to/troubleshoot/peering.md).
```

This page is about peering, and so this section is already about troubleshooting peering. So the additional troubleshoot peering page is for issues not fixed by these first steps. Alternatively, we could consolidate this section into the troubleshoot peering page and just link to that.

Reviewer 2: Yes, it would probably be best practice to add this section to the troubleshooting page and just link to the separate page.
